lib/_debug_agent.js

```diff
@@ -37,6 +37,9 @@ exports.start = function start() {
     });
 
     agent.close();
+    agent.clients.forEach(function(client) {
+      client.destroy();
+    });
```

Comment:
Tiny style nit: we're moving towards arrow functions in core.  Not really wrong though, just a little at odds with the "house style."